src/nodule.js

```diff
@@ -17,16 +12,22 @@ export default class Nodule {
         this.loaders = loaders || [];
```

Comment:
leaving `this.loaders` name unchanged to support backwards compatibility. Likely this is only used here but if this has been used in other places we don't want to change behaviour.